pastepwn/analyzers/regexanalyzer.py

```diff
@@ -12,10 +12,17 @@ def __init__(self, actions, regex, flags=0):
         super().__init__(actions)
         self.regex = re.compile(regex, flags)
 
+    def verify(self, results):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/d-Rickyy-b/pastepwn/pullRequest?prid=4945865)